src/openforms/payments/models.py

```diff
@@ -122,6 +122,12 @@ class SubmissionPayment(models.Model):
         default=PaymentStatus.started,
         help_text=_("Status of the payment process in the configured backend."),
     )
+    provider_payment_id = models.CharField(
+        _("provider payment ID"),
+        max_length=128,
```

Comment:
is there a source of the possible lengths? if it's an educated guess, can you document this "magic number"?

Reply:
Not really, I'm not sure what would be an appropriate number here tbh :grimacing:

Reply:
please add a comment that this length is a hopeful guess that payment IDs will not be longer, but if other providers are implemented, we might need to switch this to `models.TextField` at some point